python-qa-tools/utils/txs.py

```diff
@@ -0,0 +1,59 @@
+import os,json, sys
+from modules.bank.tx import tx_send, tx_sign
+from utils.commands import exec_command
+
+DAEMON = os.getenv('DAEMON')
+DAEMON_HOME = os.getnev('DAEMON_HOME')
+DENOM = os.getenv('DENOM')
+CHAINID = os.getenv('CHAINID')
+HOME = os.getnev('HOME')
+
+def fetch_seq_no(address, RPC):
+    command = f"{DAEMON} q account {address} --node {RPC} --output json"
+    seq, seqerr = exec_command(command)
+    if len(seqerr):
+        return False, seqerr
+    seq = json.loads(seq)
+    return True, int(seq['sequence'])
+
+def unsigned_tx(from_address, to_address, file_name):
+    try:
+        status, unsignedTx = tx_send(from_address, to_address, gas = 500000, unsigned = True, output = "json")
```

Comment:
Let's only use `--generate` for generating unsigned tx
```suggestion
        status, unsignedTx = generate_tx_send(from_address, to_address, gas = 500000, unsigned = True, output = "json")
```